src/revlanguage/workspace/RevVariable.cpp

```diff
@@ -105,7 +74,7 @@ RevVariable::RevVariable(const RevVariable &v) :
 RevVariable::~RevVariable( void )
 {
     
-    if ( is_reference_var == false && rev_object != NULL )
+    if ( not isReferenceVariable() and rev_object != NULL )
```

Comment:
I get that you can do this now, but it is stylistically a bit out of sync with other rev code. Not that I think it's a big deal---this is perfectly readable.

Reply:
I personally prefer `if (condition_holds)` over `if ( condition_holds == true )` since
* you could also write `if ( (condition_holds == true) == true)`
* people don't usually write `if ( (i > 1) == true)`
* the former seems closer to the way people talk "If the condition holds, then..."
I'm not aware of any other project that compares boolean values to `true` to get another boolean.

Anyway, that's my 2 cents.

Reply:
I guess this raises the question about the extent to which
* we should enforce coding style when reviewing.
* we should treat common patterns in revbayes as a style guide.

I tend not to enforce my coding style when reviewing, for better or worse.
Joelle also said that if we want to have a style guide, we should write one and get agreement to it-- not say that common patterns in revbayes source constitute a style guide.

Reply:
To be clear I don't think this is a big deal, I was just mentioning it because I had not seen it before. For me what matters is that the code is readable and maintainable, which this certainly is.